openlibrary/utils/sentry.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import sentry_sdk
+import web
+from sentry_sdk.utils import capture_internal_exceptions
+
+from openlibrary.utils import get_software_version
+
+
+def header_name_from_env(env_name):
+    # type: (str) -> str
+    """
+    Convert an env name as stored in web.ctx.env to a "normal"
+    header name
+
+    >>> header_name_from_env('HTTP_FOO_BAR')
+    'foo-bar'
+    >>> header_name_from_env('CONTENT_LENGTH')
+    'content-length'
+    """
+    header_name = env_name
+    if env_name.startswith('HTTP_'):
+        header_name = env_name[5:]
+    return header_name.lower().replace('_', '-')
+
+
+def add_web_ctx_to_event(event, hint):
```

Comment:
This is a superpower!!

Reply:
I haven't been able to get it to work for handled events in coverstore, but it's showing up for a lot of things! 🎉